pkg/sdk/tasks_def.go

```diff
@@ -114,15 +114,18 @@ var TasksDef = g.NewInterface(
 			OptionalQueryStructField(
 				"Set",
 				g.QueryStruct("TaskSet").
-					OptionalIdentifier("Warehouse", g.KindOfT[AccountObjectIdentifier](), g.IdentifierOptions().SQL("WAREHOUSE")).
+					OptionalIdentifier("Warehouse", g.KindOfT[AccountObjectIdentifier](), g.IdentifierOptions().Equals().SQL("WAREHOUSE")).
```

Comment:
we can add validation g.ValidIdentifierIfSet for Warehouse identifier

Reply:
It's not needed now because an exactlyOneValueSet underneath uses the valueSet, which validates the correctness of identifiers. With such validation, we could never get to its branch to test it.